### DISCUSSION TITLE
documentation: mbr syslinux debian install

### DISCUSSION THREAD

**Discussion by QuelonaSec** [zbm-dev/zfsbootmenu#580] [Feature Requests]
I'd like to install debian bookwork on an old HP Microserver Gen8 which doesn't support UEFI booting.

It would be very helpful to document the mbr syslinux way to go. I fear I personally won't be able to successfully merge void syslinux mbr and debian UEFI. Thanks in advance! I could help testing afterwards. :)

**Comment by ofthesun9** [comment]
Hello, 
The chapter 3.1.2 Manually [of this archlinux wiki](https://wiki.archlinux.org/title/Syslinux) is very helpful

I installed extlinux bootloader and zfsbootmenu on an external thumbdrive, as my internal hard drives are fully dedicated to my zfs pool
